server/models/message.coffee

```diff
@@ -555,11 +556,17 @@ module.exports = class Message extends cozydb.CozyModel
         async.eachSeries attachments, (att, cb) =>
             # WEIRDFIX#1 - some attachments name are broken
             # WEIRDFIX#2 - some attachments have no buffer
+            # WEIRDFIX#3 - use a stream instead of a buffer
             # att.name = att.name.replace "\ufffd", ""
-            # attachBinary need a path attributes
             att.buffer ?= new Buffer 0
-            att.buffer.path = encodeURI att.name
-            @attachBinary att.buffer, name: att.name, cb
+            stream = simplebufferstream att.buffer
```

Comment:
@aenario if you think it make more sense to do that in cozydb, I can move this hack to cozydb.